internal/gke/deploy.go

```diff
@@ -148,6 +163,7 @@ func PrepareRollout(ctx context.Context, config CloudConfig, cfg *config.GKEConf
 		once.Do(func() {
 			firstErr = err
 			cancel()
+			fmt.Fprintln(os.Stderr, "Stopping with error:", firstErr)
```

Comment:
Debugging statement?

Reply:
No, I wanted to leave this in there because otherwise the error won't get printed until the build completes, which may be a while.